cmd/maya-exporter/app/command/exporter.go

```diff
@@ -31,10 +34,25 @@ func Initialize(options *VolumeExporterOptions) string {
 // info.
 
 // StartMayaExporter starts an HTTP server that exposes the metrics on
-// "/metrics" endpoint.
+// <clusterIP>:9500/metrics endpoint and for getting the metrics in json
+// the "type=json" action can be used. e.g <clusterIP>:9500/metrics/?type=json
 func (options *VolumeExporterOptions) StartMayaExporter() error {
 	glog.Info("Starting http server....")
 	http.Handle(options.MetricsPath, promhttp.Handler())
+	http.HandleFunc(options.MetricsPath+"/", func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.RawQuery == "type=json" {
```

Comment:
should we also handle the case if RawQuery is not as expected and return StatusNotFound ?

Reply:
yes we can handle that as bad request